### ISSUE TITLE
Error Unknown volume for chapter xx in syosetu

### ISSUE THREAD

**Issue by Arisa43fps** [dipu-bd/lightnovel-crawler#2082]
## Let us know

**Novel URL**:     https://ncode.syosetu.com/n3652gg/ 
**App Location**:    PIP | EXE | Discord  
**App Version**:   3.2.10  

## Describe this issue

I tried to download Japanese novels from the syosetu site and got an error.
I've attached the log below.

```
Retrieving novel info...
C:\Users\Arisa43fps\AppData\Roaming\Python\Python310\site-packages\bs4\builder\__init__.py:545: XMLParsedAsHTMLWarning: It looks like you're parsing an XML document using an HTML parser. If this really is an HTML document (maybe it's XHTML?), you can ignore or filter this warning. If it's XML, you should know that using an XML parser will be more reliable. To parse this document as XML, make sure you have the lxml package installed, and pass the keyword argument `features="xml"` into the BeautifulSoup constructor.
  warnings.warn(

 ! Error: Unknown volume for chapter 58
<class 'AssertionError'>
File "C:\Users\Arisa43fps\AppData\Roaming\Python\Python310\site-packages\lncrawl\bots\console\integration.py", line 85, in _download_novel
    self.app.get_novel_info()
  File "C:\Users\Arisa43fps\AppData\Roaming\Python\Python310\site-packages\lncrawl\core\app.py", line 135, in get_novel_info
    format_novel(self.crawler)
  File "C:\Users\Arisa43fps\AppData\Roaming\Python\Python310\site-packages\lncrawl\core\novel_info.py", line 71, in format_novel
    __format_chapters(crawler, vol_id_map)
  File "C:\Users\Arisa43fps\AppData\Roaming\Python\Python310\site-packages\lncrawl\core\novel_info.py", line 52, in __format_chapters
    assert vol_index >= 0 and vol_index < len(
```

**Comment by zGadli** [comment]
Please specify the App location in your issue properly. Did you clone this repo to your local machine and get this error? If yes, then mention PIP. If you installed .exe file for this repo then mention EXE.

**Comment by Arisa43fps** [comment]
@zGadli For that issue, I'm also having the same issue in PIP, EXE, and Discord bots. Maybe you can try it yourself. To run from source I haven't tried it.

**Comment by Arisa43fps** [comment]
@zGadli Now I can download from the Syosetu site, but there is an error when I open the file which says "Failed to download chapter body"
![image](https://github.com/dipu-bd/lightnovel-crawler/assets/87550572/850b7cd1-4591-44de-a1ee-e0e569b0114d)

**Comment by Arisa43fps** [comment]
Here's the log
[syosetu.log](https://github.com/dipu-bd/lightnovel-crawler/files/12739533/syosetu.log)

**Comment by Arisa43fps** [comment]
Now the problem has been resolved by updating to version 3.3.1
